core/lines/line_commands.talon

```diff
@@ -0,0 +1,7 @@
+# FIXME: TEMPORARY: Once https://github.com/talonhub/community/pull/1449 is merged, this file shoulid be removed
```

Comment:
I would put rename that file into a `core/edit/edit.talon` new file to limit the number of dedicated files until we really need to split, especially that this is temporary

Reply:
we agreed to do this even though in community it's in a different file. The reasoning behind it is that in community it should probably go into the same edit.talon